TracingLauncher/Program.cs

```diff
@@ -2,15 +2,26 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
-using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
+using System.Timers;
+using Timer = System.Timers.Timer;
 
 namespace TracingLauncher
 {
     class Program
     {
+        private const int FlushPeriod = 1000;
```

Comment:
One tick is 100 nanosecond, so that's still a very short amount of time. Did you mean to use milliseconds instead, such that flush is every second?

Reply:
Good catch, meant to check for milliseconds. Fixed.